src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -181,6 +184,12 @@ def __init__(self, cli_ctx, command, expect_failure=False):
         self.applog = ''
         self.command_coverage = {}
 
+        if os.environ.get(ENV_COMMAND_COVERAGE, None) == 'True':
```

Comment:
Should be enough just to check for the presence of the env var right?
People may try `1`, `true` etc. so I'd consider simplifying this check.